.github/workflows/tagbot.yml

```diff
@@ -0,0 +1,30 @@
+name: Tagbot
+on: [pull_request]
+
+jobs:
+  tagbot:
+    runs-on: ubuntu-20.04
```

Comment:
Can we use a more recent OS, this is likely to be deprecated soon?

Reply:
Done. I also reworked some code in order to limit the number of diffs. Will make some PRs on my own branch for testing to ensure i didn't mess it up, I'll comment again with links when I've had time to do that